src/ast/sql-query/sql-query.ts

```diff
@@ -277,6 +277,73 @@ export class SqlQuery extends BaseAst {
     });
   }
 
+  hasGroupByForColumn(column: string): boolean {
+    return this.getGroupByColumns().includes(column);
+  }
+
+  getGroupByColumns(): string[] {
+    if (!this.groupByClause) {
+      return [];
+    }
+
+    const groupByColumnNames: string[] = [];
+    const groupByColumns = this.groupByClause.groupBy.map(part => part.getBasicValue());
+    this.columns.columns.forEach((column, index) => {
+      index = index + 1;
+      if (
+        arrayContains(column.getBasicValue(), groupByColumns) ||
+        arrayContains(String(index), groupByColumns)
+      ) {
+        groupByColumnNames.push(column.getBasicValue());
+      }
+    });
+
+    return groupByColumnNames;
+  }
```

Comment:
add an empty line between methods please

Reply:
should this be detected by a linter? Do we have one configured? If not, perhaps @renevan10 can add it?

Reply:
Would this mean adding a configuration in the .eslintrc.js file?